modules/apps/portal-template/portal-template-freemarker/src/test/java/com/liferay/portal/template/freemarker/internal/RestrictedLiferayObjectWrapperTest.java

```diff
@@ -243,12 +247,18 @@ public void testRestrictedMethodNames() throws Exception {
 		_testRestrictedMethodNames(liferayFreeMarkerStringModel, "Name");
 		_testRestrictedMethodNames(liferayFreeMarkerStringModel, "getName");
 		_testRestrictedMethodNames(liferayFreeMarkerStringModel, "getname");
+		_testRestrictedMethodNames(liferayFreeMarkerStringModel, "dslQuery");
 
 		Assert.assertEquals(
 			"Denied access to the toString method in class " +
 				TestLiferayMethodObject.class,
 			liferayFreeMarkerStringModel.getAsString());
 
+		Assert.assertEquals(
+			"Denied access to the interface methods in class " +
+				TestLiferayMethodObject.class,
+			liferayFreeMarkerStringModel.getDeniedInterfaceMethods());
```

Comment:
Also, if we only focus on this commit
I don't think there is getDeniedInterfaceMethods() in liferayFreeMarkerStringModel
If someone cherry pick this commit only, then they will have compile issue